python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -508,7 +510,15 @@ def updateBlendRecords(blendData, catalog, modelPsf, observedPsf, redistributeIm
             # Set the flux at the center of the model
             peak = heavy.peaks[0]
             img = heavyFootprintToImage(heavy, fill=0.0)
-            sourceRecord.set("deblend_peak_instFlux", img.image[Point2I(peak["i_x"], peak["i_y"])])
+            try:
+                sourceRecord.set("deblend_peak_instFlux", img.image[Point2I(peak["i_x"], peak["i_y"])])
+            except Exception:
+                srcId = sourceRecord.getId()
+                x = peak["i_x"]
+                y = peak["i_y"]
+                warnings.warn(f"Source {srcId} at {x},{y} could not set the peak flux with error:")
```

Comment:
Would it make more sense to use LSST's logging rather than warnings?

Reply:
I agree this looks more like a `log.warning()` -- warnings to get converted to log messages but they won't be found as easily because the wrong logger will be used.

Reply:
Does it work to import the logger using
```
import logging
logger = logging.getLogger("lsst.meas.extensions.scarlet")
logger.warning("Log message here")
```
or do I have to pass it the logger from a task?

Reply:
I thought measurement plugins had access to a logger so does `logging.getLogger(self.logName)` work to get the logger? That seems to work in some meas_extensions packages.

If that doesn't work use `logger = logging.getLogger(__name__)` so you don't have to hard code the full name in there.